src/js/protocols/stm32usbdfu.js

```diff
@@ -225,34 +230,34 @@ STM32DFU_protocol.prototype.getInterfaceDescriptors = function (interfaceNum, ca
             return;
         }
 
-	var interfaceID = 0;
-	var descriptorStringArray = [];
-	var getDescriptorString = function () {
-		if(interfaceID < config.interfaces.length) {
-			self.getInterfaceDescriptor(interfaceID, function (descriptor, resultCode) {
-				if (resultCode) {
-				    callback([], resultCode);
-				    return;
-				}
-				interfaceID++;
-				self.getString(descriptor.iInterface, function (descriptorString, resultCode) {
-					if (resultCode) {
-					    callback([], resultCode);
-					    return;
-					}
-					if (descriptor.bInterfaceNumber == interfaceNum) {
-						descriptorStringArray.push(descriptorString);
-					}
-					getDescriptorString();
-				});
-			});
-		} else {
-			//console.log(descriptorStringArray);
-			callback(descriptorStringArray, 0);
-			return;
-		}
-	}
-	getDescriptorString();
+    var interfaceID = 0;
+    var descriptorStringArray = [];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'var' statement.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626377)